### ISSUE TITLE
Possible Chisel Issue

### ISSUE THREAD

**Issue by VintageCiv** [anegostudios/VintageStory-Issues#1623]
**Game Version:** 16
**Platform:** Windows  
**Modded:** Yes
![image](https://user-images.githubusercontent.com/96968356/150476849-4a2c027d-1031-455e-af65-1a0cc27ab4b9.png)

**SP/MP:** Multiplayer

### Description
Players around the offending block are all stuck in death loop, error message is as follows:

### How to reproduce

Unsure how to, we are still troubleshooting that ourselves

### Expected behavior
See above

### Screenshots
![image](https://user-images.githubusercontent.com/96968356/150479060-3603d6b9-176b-4f2b-9f29-05f73c7f7a88.png)

### Logs
```
21.1.2022 00:31:15 [Error] Found closing tag <string> at position 1340 but <strong> should be closed first. See debug log for full text.
21.1.2022 00:31:21 [Fatal] Caught unhandled exception in thread 'networkproc'. Exiting game.
21.1.2022 00:31:21 [Fatal] System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at Vintagestory.GameContent.BlockEntityMicroBlock.CreateMesh(ICoreClientAPI coreClientAPI, List`1 voxelCuboids, Int32[] materials, BlockPos posForRnd)
   at Vintagestory.GameContent.BlockEntityMicroBlock.FromTreeAttributes(ITreeAttribute tree, IWorldAccessor worldAccessForResolve)
   at Vintagestory.Client.NoObf.ClientChunk.PreLoadBlockEntitiesFromPacket(_XyJ0AvcR2vjLZ2mfN5MTsLhbkwg[] blockEntities, Int32 blockEntitiesCount, ClientMain game)
   at Vintagestory.Client.NoObf.ClientWorldMap.LoadChunkFromPacket(_lOK0xxasAcp2QoIIaRxSlTsrpfi p)
   at Vintagestory.Client.NoObf.SystemNetworkProcess.ProcessInBackground(_hpqb4cNQuSfxZLdFacKLPiYYWAN packet)
   at Vintagestory.Client.NoObf.SystemNetworkProcess.TryReadPacket(Byte[] data, Int32 dataLength)
   at Vintagestory.Client.NoObf.SystemNetworkProcess.NetworkProcess()
   at Vintagestory.Client.NoObf.ClientThread.Update()
   at Vintagestory.Client.NoObf.ClientThread.Process()
```
[client-main.txt](https://github.com/anegostudios/VintageStory-Issues/files/7910847/client-main.txt)

**Comment by Aledark789** [comment]
One error is on the handbook of temporal stability (guide). Search keyword name string instead of strong with the <> vtml tags

**Comment by copygirl** [comment]
Yeah the handbook issue is tracked in #1608. Not related to this bug.

**Comment by tyronx** [comment]
Can you send me or discord DM me coords of the crashing site - then i'll connect to repro it

**Comment by DArkHekRoMaNT** [comment]
Fixed in [v1.16.4](https://www.vintagestory.at/blog.html/news/homesteading-part-2-combat-update-stabler-er-finetune-1164-rc1-r316/)
